features/all-positions/AllPositions.tsx

```diff
@@ -61,80 +59,114 @@ const AllPositions = () => {
     collateral: BigNumberish,
     tokens: BigNumberish
   ) => {
-    if (!latestPrice) return null;
+    if (latestPrice === 0) return null;
     const tokensScaled = Number(tokens) * Number(latestPrice);
     return (Number(collateral) / tokensScaled).toFixed(4);
   };
 
-  return (
-    <Box>
+  if (
```

Comment:
@chrismaree @mrice32 I think this is a pretty clean and organized way to handle the the null-check for rendering logic (#68) and I'm going to apply it across all components in this PR. Here's the idea:

Import and load all containers first, and then define any helper functions.

Only render the main page if all "data" from containers and other related variables are deemed valid (i.e. not `null` or `undefined`). This results in the big `if` statement before the main render `return()`.

If any of the "data" is not ready, then show the fallback JSX component. This way all of the main rendering logic can assume that data is valid.